app/src/main/java/de/westnordost/streetcomplete/quests/surface/DetailRoadSurfaceForm.kt

```diff
@@ -0,0 +1,105 @@
+package de.westnordost.streetcomplete.quests.surface
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+import androidx.appcompat.app.AlertDialog
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.quests.AGroupedImageListQuestAnswerFragment
+import de.westnordost.streetcomplete.quests.OtherAnswer
+import de.westnordost.streetcomplete.util.TextChangedWatcher
+import de.westnordost.streetcomplete.view.Item
+import de.westnordost.streetcomplete.quests.surface.Surface.*
+
+class DetailRoadSurfaceForm  : AGroupedImageListQuestAnswerFragment<String, DetailSurfaceAnswer>() {
+
+    override val topItems get() =
+        if (osmElement!!.tags["surface"] == "paved")
+            listOf(ASPHALT, CONCRETE, SETT, PAVING_STONES, WOOD, GRASS_PAVER).toItems()
```

Comment:
Why not use `pavedSurfaces().toItems()` etc.?

Reply:
Because it is 6 surfaces (fitting nicely in two rows), while `pavedSurfaces()` have currently 8 of them. Currently `UNHEWN_COBBLESTONE` and `METAL` are not listed.

I would be also perfectly fine with listing all available paved surfaces here.

Reply:
Let's list all of them. Some months ago, we removed most/all of the "Show more" stuff.

Reply:
switched to one big uniform list